.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea/
```

Comment:
not quite sure what this does.

Reply:
This was to ensure my Intellij IDEA files do not get committed. Can remove but suspect others may face this too.

Reply:
ah gotcha, it should be worth adding then